FLEDGE.md

```diff
@@ -643,7 +643,8 @@ Reports are only sent and most interest group state changes (e.g. updating `prev
 
 Once the winning ad has rendered in its Fenced Frame, the seller and the winning buyer each have an opportunity to perform logging and reporting on the auction outcome.  The browser will call one reporting function in the seller's auction worklet and one in the winning buyer's bidding worklet.
 
-_As a temporary mechanism,_ these reporting functions will be able to send event-level reports to their servers.  These reports can include contextual information, and can include information about the winning interest group if it is over an anonymity threshold.  This reporting will happen synchronously, while the page with the ad is still open in the browser.
+_As a temporary mechanism,_ these reporting functions will be able to send event-level reports to their servers.  These reports can include contextual information, and can include information about the winning interest group if it is over an anonymity threshold.
+The worklet functions `reportWin()` and `reportResult()` may call a `sendReportTo()` API which takes a single string argument representing a URL. The URL will is fetched after the reporting worklet completes.  This fetch will happen synchronously, while the page with the ad is still open in the browser.
```

Comment:
Could you also add any constraints on the `sendReportTo()` call? Example: It can be called only once from the reporting functions.

Reply:
Done